apps/utils/helpers.py

```diff
@@ -0,0 +1,20 @@
+from django.template.response import TemplateResponse
+from django.urls import reverse
+
+from apps.generics.help import render_help_with_link
+
+
+def generic_home(request, team_slug: str, title: str, table_url_name: str, new_url: str):
+    return TemplateResponse(
+        request,
+        "generic/object_home.html",
+        {
+            "active_tab": title.lower(),
+            "title": title,
+            "title_help_content": render_help_with_link("", title.lower()),
```

Comment:
Using `title.lower()` seems risky since it would be easy for someone to change the title not realizing it also impacts the help text. It also makes it hard to find usages of a specific help text key.